shared/common-adapters/emoji.desktop.js

```diff
@@ -10,9 +10,22 @@ import type {Props} from './emoji'
 
 const backgroundImageFn = (set: string, sheetSize: string) => emojiSet
 
+const skinToneRegex = /:([^:]*):(:skin-tone-\d:)?/i
+
+const isValidEmoji = (fullEmoji: string) => {
```

Comment:
we should probably document the bug we're working around here, and/or file an upstream bug so that we can avoid it in the future.